install/kubernetes/helm/istio/values.yaml

```diff
@@ -444,3 +444,8 @@ global:
   # This field is set to false by default, so 'helm template ...'
   # will ignore the helm test yaml files when generating the template
   enableHelmTest: false
+
+  # Whether to use the proxy protocol, useful for capturing the real client's IP address when a load balancer
+  # is in front of envoy, like an ELB.
+  # See https://www.envoyproxy.io/docs/envoy/latest/configuration/listener_filters/proxy_protocol
+  proxyProtocol: false
```

Comment:
Why in global ? Seems a pilot-specific feature

Reply:
agree, will set this in the pilot section

Reply:
ok, this is moved into the pilot chart's value.yaml